src/components/Form/FormSection.jsx

```diff
@@ -14,11 +14,12 @@ import { useTheme } from '@mui/material/styles';
  * @name FormSection
  * @param {object} Props - A React prop that consists of
  * that consist of title and children (see {@link formSectionProps})
+ * @param {string} Props.id - The `id` attribute of the element containing the `title`
  * @param {string} Props.title - Title of form section
  * @param {React.ReactElement} Props.children - JSX Element of the wrapped form
  * @returns {React.JSX.Element} - The FormSection Component
  */
-const FormSection = ({ title, children }) => {
+const FormSection = ({ title, id, children }) => {
```

Comment:
Just a nitpick, but think we could use something like `htmlId` for the prop name as to not get confused for where the `id` attribute is attributed to (the component itself or an element within the component).